src/Helpers/StateHelper.php

```diff
@@ -2,17 +2,27 @@
 
 namespace Sammyjo20\Saloon\Helpers;
 
-use Illuminate\Support\Str;
-
 class StateHelper
 {
     /**
      * Generate a random string for the state.
      *
      * @return string
+     * @throws \Exception
      */
     public static function createRandomState(): string
     {
-        return Str::random(32);
+        $string = '';
+        $length = 32;
+
+        while (($len = strlen($string)) < $length) {
+            $size = $length - $len;
+
+            $bytes = random_bytes($size);
+
+            $string .= substr(str_replace(['/', '+', '='], '', base64_encode($bytes)), 0, $size);
+        }
+
+        return $string;
```

Comment:
Same here, I kind of wish we had the same functionality as: https://github.com/illuminate/support/blob/master/Str.php#L726